actions/commitizen-cz-cli/README.md

```diff
@@ -0,0 +1,28 @@
+# Commitizen (cz-cli)
+
+https://github.com/commitizen/cz-cli
+
+## Simple configuration
+
+Under the default configuration, Commitizen will pull its configuration from commitlint config files, as there is a high likelihood both tools will be used together. The below sample configuration will apply the [@commitlint/config-conventional](https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-conventional) standard for both Commitizen and commitlint.
+
+1. `echo "{"path": "@commitlint/cz-commitlint"}" > .czrc`
```

Comment:
```suggestion
1. `echo '{"path": "@commitlint/cz-commitlint"}' > .czrc`
```
When I tried this my shell ate the inner quotes. This should fix it. Or maybe just use code blocks for all of these that people can copy-paste? ie:
Place the following in `.czrc`:
```
"path": "@commitlint/cz-commitlint"}
```